### PR TITLE
Feat: Add new remote company - Wolverine Trading

### PR DESCRIPTION
**Pull request by tonyruiz-o**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -722,6 +722,7 @@ Name | Website | Region
 [Wizeline](/company-profiles/wizeline.md) | https://www.wizeline.com/ | Worldwide
 [Wolfram](/company-profiles/wolfram.md) | https://www.wolfram.com | Worldwide
 [Wombat Security Technologies](/company-profiles/wombat-security.md) | https://www.wombatsecurity.com/ | USA
+[Wolverine Trading](/company-profiles/wolverine-trading.md) | https://www.wolve.com/ | USA, UK
```
Comment (KieranRobson) [inline README.md:725]: This should be above `Wombat Security Technologies`

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Hi @truizormeno!

One quick change to be made!

File: README.md (hunk 1 above)
Comment (truizormeno) [reply README.md:725]: Oops, will fix now.

**Review by KieranRobson — APPROVED** [review]
Looks good!

**Comment by dougaitken** [comment]
Thank you for the contribution and your patience @tonyruiz-o

**Pull request merged**